packages/fx-core/src/plugins/resource/aad/utils/common.ts

```diff
@@ -63,4 +64,30 @@ export class Utils {
     const tenantId: string = (tokenObject as any).tid;
     return tenantId;
   }
+
+  public static getAndMoveInput(ctx: PluginContext): boolean {
+    if (!isMultiEnvEnabled()) {
+      const skip = ctx.config.get(ConfigKeys.skip) as boolean;
+      return skip;
+    }
+
+    const objectId = ctx.envInfo.config.auth?.objectId;
+    const clientId = ctx.envInfo.config.auth?.clientId;
+    const oauth2PermissionScopeId = ctx.envInfo.config.auth?.accessAsUserScopeId;
+    const clientSecret = ctx.envInfo.config.auth?.clientSecret;
+    const skip = ctx.envInfo.config.auth?.[ConfigKeys.skip];
```

Comment:
According to the env config schema: https://aka.ms/teamsfx-env-config-schema, we don't have the `skip` config key in the new env config.

Reply:
I will update the schema. Thx!

Reply:
Will fix in another pr, thx!

Reply:
OK. So can you remove this line which is reading `skip` property from env config? You can consolidate the skip aad provision/configuration logic in another PR.